api/package.json

```diff
@@ -8,7 +8,7 @@
     "eslint": "eslint ./src",
     "flow:check": "flow check ./src/",
     "flow:build": "flow-remove-types ./src/ -d ./dist/ --all --pretty",
-    "build": "yarn eslint && yarn flow:check && yarn flow:build",
```

Comment:
I don't think this is necessary any more. I expect it was due to the top-level import from `dist` in the test file, which is no longer present.

Reply:
OK, great. It definitely doesn't seem like the right thing to do in any case.